api/household/householdRouter.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const DB = require('../utils/db-helper');
+// const DB = require('../utils/db-helper');
```

Comment:
If we _know_ we won't be using the old `db-helper` code, we should probably remove the code outright, not comment it out.

Reply:
Kind of like what you did in `serviceEntriesRouter.js`.